modules/ROOT/pages/osv2-faq.adoc

```diff
@@ -106,6 +106,8 @@ Mule 3 example XML flow:
 
 No, but applications running on-premises can use the Object Store v2 REST API to store data in Object Store v2 and retrieve it.
 
+NOTE: There must be an Object Store application running in the MuleSoft cloud for on-premises applications to access so that they can store data to and retrieve data from the object store. 
```

Comment:
This is not a practice that we want to encourage.  OSv2 is for Cloudhub mules.  We're not really advertising it as a way for on-prem mules to store data.  I don't think we should add this.

Reply:
ok